Create transparent-proxy.adoc (#282)
PaloAltoNetworks/prisma-cloud-docs: cns/saas/admin_guide/start/enforcer/transparent-proxy.adoc

=== Enable non-transparent proxy support

> Enforcers require access to Prisma Cloud in order to send telemetry data and receive updates and in some environments this can only be achieved through a non-transparent proxy.

Reviewer: ```suggestion
Enforcers require access to the Prisma Cloud Console to send telemetry data and receive updates. In some environments, you can only achieve this through a non-transparent proxy.
```

> Enforcers support adding a proxy endpoint during install, in order to support such use cases.

Reviewer: ```suggestion
Enforcers support adding a proxy endpoint during installation to support this use case.
```

[NOTE]

> TLS terminating proxies are not supported.

Reviewer: ```suggestion
====
TLS terminating proxies are not supported.
====
```

> During the Enforcer installation, expand the Advanced option and add the proxy information, as follows:

Reviewer: ```suggestion
During the Enforcer installation, expand the *Advanced* option and add the following proxy information.
```

> * Proxy Address — IP address or fully qualified domain name (FQDN) of the proxy server, alongside the protocol and port information: example http://proxy.example.com:8080

Reviewer: ```suggestion
* *Proxy Address:* IP address or fully qualified domain name (FQDN) of the proxy server including the protocol and port information. For example: `http://proxy.example.com:8080`
```

> * Proxy Credential— User and Password credentials for proxies that require authentication (optional)

Reviewer: ```suggestion
* *Proxy Credential:* Username and password for proxies that require authentication. (Optional)
```

> * Proxy Server CA — When the proxy server requires a private CA certificate to be used during connection (optional). Upload the proxy certificate in .pem file format.

Reviewer: ```suggestion
* *Proxy Server CA:* If your proxy server requires connections to use a private Certificate Authority (CA) certificate. (Optional). Upload the proxy certificate using the `.pem` file format.
```
